Yearn-agents/InstantWithdrawl/src/blockchain.ts

```diff
@@ -0,0 +1,32 @@
+import path from "path";
+import dotenv from "dotenv";
+const ganache = require("ganache-cli");
+dotenv.config();
+
+const FORK_TESTS_CACHE_PATH = path.join(
+  __dirname,
+  "..",
+  "provider",
+  ".hardhat_node_test_cache"
+);
+
+export const DEFAULT_HARDFORK = "byzantium";
+
+export default function runServer(accounts: Array<string>) {
```

Comment:
I don't think using this `server` is the best approach. I think using `ganache.provider` is better.